hdlConvertor/toVhdl.py

```diff
@@ -656,9 +651,12 @@ def print_hdl_import(self, o):
         self.print_doc(o)
         w = self.out.write
         lib_name = o.path[0]
+        #rhinton:: conversion from Verilog probably doesn't create an explicit
```

Comment:
True, but lets keep verilog AST in verilog format and have an explicit transformation which converts verilog AST to vhdl (adds library definitions, converts replicative operators to concatenation, event dependencies in sensitivity list to a event operators in process body...)

For now let it like it is at the end of the next month I will finally have time to publish the toVerilog/VHDL rework (https://github.com/Nic30/hdlConvertor/issues/51)

Reply:
Sounds good.  Then this method shouldn't have to insert a library declaration since the new parser recognizes them from the original HDL.  Would you rather keep the current auto-library functionality (perhaps with a warning), or insist that the user or converter code includes the proper library declaration (with or without error checking -- and the latter could get hairy)?

I'm glad to hear the language conversion rework is close.

Reply:
> insist that the user or converter code includes the proper library declaration

With error checking on use statements.

Reply:
For the moment I removed the auto-library code.  It seems like this error-checking code belongs somewhere else, not in the "emitter" code.  Do you have any error checking or elaboration-type code currently?

Reply:
I do have error checking code in "emitter" and "translator". Emitter check if it supports such a object. And translator check if the object is in correct format (type checking is part of translator and is ignored in emitter).